tests/test_lab_diff.py

```diff
@@ -33,102 +33,91 @@ def test_diff(self):
         tracked_file = "compositional_skills/tracked/qna.yaml"
         self.taxonomy.create_untracked(untracked_file)
         self.taxonomy.add_tracked(tracked_file)
-        runner = CliRunner()
-        with runner.isolated_filesystem():
-            result = runner.invoke(
-                lab.diff,
-                [
-                    "--taxonomy-base",
-                    TAXONOMY_BASE,
-                    "--taxonomy-path",
-                    self.taxonomy.root,
-                ],
-            )
-            self.assertIn(untracked_file, result.output)
-            self.assertNotIn(tracked_file, result.output)
-            self.assertEqual(result.exit_code, 0)
+        result = CliRunner().invoke(
```

Comment:
Does it matter that there are now 2 CliRunners? One from the fixture for isolation and one in the test method for invoke. Perhaps the fixture could pass the runner to the test method so that it could be used for invoke?

Reply:
I don't think it matters. For what I see in the context manager, it just changes the directory into a temp directory; then switches back on __exit__. In theory, we can even replace this isolated_filesystem context manager that depends on the runner with something else, runner is just a container for "useful utilities" in this context.